AWS marketplace requirements - Critical update! (#4139)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.2/choose-infrastructure/aws/index.md

enterprise: false
---

> The following diagrams show the two different ways you can implement Konvoy on AWS, DKP Essentials and DKP Enterprise.

Reviewer: Not sure if Essentials or Essential.

Author: That makes two of us, I've seen it both ways! I thought I was valid with "s", but maybe not!

> The following diagrams show the two different ways you can implement Konvoy on AWS, DKP Essentials and DKP Enterprise.

Author: ```suggestion
The following diagrams show the two different ways you can implement Konvoy on AWS, DKP Essential and DKP Enterprise.
```

This diagram shows the granular detail of a single Kubernetes cluster running in AWS Cloud:

> ![DKP Essentials](/dkp/konvoy/2.2/img/DKP_essentials.png)

Author: ```suggestion
![DKP Essential](/dkp/konvoy/2.2/img/DKP_essentials.png)
```

This diagram shows a higher-level view of DKP Enterprise, and assumes a multi-cluster environment, where each cluster might look like the single cluster example above:

![DKP Enterprise](/dkp/konvoy/2.2/img/DKP_enterprise.png)

> ## AWS Pricing considerations

Author: ```suggestion
## AWS pricing considerations
```

> Deploying AWS services can incur you cost, depending on how and what you deploy. For more information, see the [AWS Pricing Calculator](https://calculator.aws/#/).

Author: ```suggestion
Deploying AWS services can incur costs to your organization, depending on how and what you deploy. For more information, see the [AWS Pricing Calculator](https://calculator.aws/#/).
```

> ## AWS Service limits

Author: ```suggestion
## AWS service limits
```

> When using DKP on AWS, you need to be aware of the possibility of errors due to AWS service limits. For more information, see  the [AWS Service Limits](https://aws.amazon.com/premiumsupport/knowledge-center/manage-service-limits/).

Author: ```suggestion
When using DKP on AWS, you need to be aware of the possibility of errors due to AWS service limits. For more information, see the [AWS Service Limits](https://aws.amazon.com/premiumsupport/knowledge-center/manage-service-limits/).
```
